Purchases/Public/Purchases.swift

```diff
@@ -1025,6 +1027,25 @@ public extension Purchases {
         purchasesOrchestrator.showManageSubscriptionModal(completion: completion)
     }
 
+    /**
+     * Presents a refund request sheet in the current window scene for
+     * the latest transaction associated with the productID
+     *
+     * - Parameter productID: The productID to begin a refund request for.
+     * - Parameter completion: A completion block that is called when the modal is closed.
+     * If the request was successful, there will be a `RefundRequestStatus`.
+     * Keep in mind the status could be `userCancelled`
+     * If the request was unsuccessful, there will be an `Error`.
+     */
+    @available(iOS 15.0, macCatalyst 15.0, *)
+    @available(watchOS, unavailable)
+    @available(tvOS, unavailable)
+    @available(macOS, unavailable)
+    @objc func beginRefundRequest(for productID: String,
```

Comment:
Do we need `@available(watchOSApplicationExtension, unavailable)` as well?

Reply:
availability for the underlying method (in the docs) is:
iOS 15.0+
iPadOS 15.0+
Mac Catalyst 15.0+
Xcode 13.0+

looks like [this](https://github.com/llvm-mirror/clang/blob/master/include/clang/Basic/Attr.td#L782-L789) is the list of available platforms...but that doesn't include macCatalyst which we use elsewhere, so i was a little lost. there's also [this](https://docs.swift.org/swift-book/ReferenceManual/Attributes.html)

i suppose we should mimic what the underlying storekit method has, which is: 
    @available(iOS 15.0, *)
    @available(watchOS, unavailable)
    @available(tvOS, unavailable)

Reply:
I think mimicking what StoreKit has is a great idea!

Reply:
here's the syntax for catalyst: 
https://github.com/RevenueCat/purchases-ios/blob/main/Purchases/Public/Purchases.swift#L121
I think StoreKit omits it because catalyst is included in iOS